Tests/PythonTests/test_account_info.py

```diff
@@ -0,0 +1,44 @@
+"""
+Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+We are making my contributions/submissions to this project solely in our
+personal capacity and are not conveying any rights to any intellectual
+property of any third parties.
+"""
+
+import pyRosetta
+
+def test_constructors():
+    player1 = pyRosetta.AccountInfo()
+
+    assert player1.email() == 'anonymous@gmail.com'
+    assert player1.nickname() == 'Anonymous'
+
+    player2 = pyRosetta.AccountInfo('1', 'name1')
+
+    assert player2.email() == '1'
+    assert player2.nickname() == 'name1'
+
+    player3 = pyRosetta.AccountInfo('2', 'name2', [])
+
+    assert player3.email() == '2'
+    assert player3.nickname() == 'name2'
+    assert player3.num_of_deck() == 0
+
+def test_deck_control():
+    player = pyRosetta.AccountInfo()
+
+    player.show_deck_list()
+    assert player.create_deck('deck1', pyRosetta.CardClass.INVALID) == False
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to False should be 'if cond is False:' or 'if not cond:'](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4593740)